packages/styles/scss/components/structured-list/_structured-list.scss

```diff
@@ -28,6 +28,17 @@
     @include focus-outline('outline');
   }
 
+  .#{$prefix}--structured-list-row
+    .#{$prefix}--structured-list-td.#{$prefix}--structured-list-icon-column-first {
+    inline-size: $spacing-07;
+    padding-inline-end: 0;
+  }
+  .#{$prefix}--structured-list-row
+    .#{$prefix}--structured-list-td.#{$prefix}--structured-list-icon-column-last {
+    inline-size: $spacing-07;
+    padding-inline-start: 0;
+  }
+
```

Comment:
Instead of adding a new class are you able to use pseduoselectors? `:first-child` and `:last-child` ?